rdr_service/api_util.py

```diff
@@ -24,6 +25,7 @@
 STOREFRONT_AND_REDCAP = [STOREFRONT, REDCAP]
 PTC_AND_HEALTHPRO = [PTC, HEALTHPRO]
 PTC_HEALTHPRO_AWARDEE = [PTC, HEALTHPRO, AWARDEE]
+PTC_HEALTHPRO_AWARDEE_CURATION = [PTC, HEALTHPRO, AWARDEE, CURATION]
```

Comment:
For this, https://precisionmedicineinitiative.atlassian.net/browse/PD-6083 since you are giving Curation it's on constant can you edit the roles to be `curation` instead of `healthpro`

Reply:
Sorry, I didn't got you... I didn't edit anything related `healthpro`

Reply:
So in the config changes:
+     "data-analytics@aou-res-curation-prod.iam.gserviceaccount.com": {
+       "clientId": "cdr",
+       "roles": [
+         "healthpro"
+       ]
+     },
Can you edit the role list to be:
+       "roles": [
+         "curation"
+       ]
And in this PR can add a test for changing the user's role to only `curation` to make sure you are getting the correct response in the test case

Reply:
Ah, I see. Yes, I will update the configuration as well.

Reply:
https://precisionmedicineinitiative.atlassian.net/browse/PD-6083?focusedCommentId=85573